lib/.eslintrc.yaml

```diff
@@ -171,8 +171,6 @@ rules:
       message: Use `const { setInterval } = require('timers');` instead of the global.
     - name: setTimeout
       message: Use `const { setTimeout } = require('timers');` instead of the global.
-    - name: structuredClone
-      message: Use `const { structuredClone } = require('internal/structured_clone');` instead of the global.
```

Comment:
```suggestion
    - name: structuredClone
      message: Use `const { structuredClone } = internalBinding('messaging');` instead of the global.
```